c2corg_ui/static/js/detailsview.js

```diff
@@ -0,0 +1,68 @@
+goog.provide('app.ViewDetailsController');
+goog.provide('app.viewDetailsDirective');
+
+goog.require('app');
+
+/**
+ * @return {angular.Directive} The directive specs.
+ */
+app.viewDetailsDirective = function() {
+  return {
+    restrict: 'A',
+    controller: 'AppViewDetailsController',
+    controllerAs: 'detailsCtrl',
+    bindToController: true,
+    link: function(el, scope, attrs) {
+      // show tabs if they have been hidden on smartphone
+      $(window).resize(function() {
+        if (window.innerWidth > 620) {
+          $('.tab, .accordion').show();
```

Comment:
If no new elements with a class `.tab` or `.accordion` are added, you could cache the selection:

    var elements = $('.tab, .accordion');
    $(window).resize(function() {
      if (window.innerWidth > 620) {
        elements.show();